firmware/robot2015/cpu/utils/FirmwareCheck/FirmwareCheck.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef FIRMWARE_CHECK_H
+#define FIRMWARE_CHECK_H
+
+#include "mbed.h"
+#include <string>
+
+extern void firmware_version(std::string&);
```

Comment:
Can we add a quick comment on here just to clarify that this checks the MBED's hardcore builtin firmware and not the version of the RoboCup firmware?  I think at some point, we'll bake the git repo's SHA checksum into the our firmware, which will be how we track our own versioning.